README.md

```diff
@@ -53,7 +56,8 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+The memory requirement would be O(n) because the stack for the parser will only contain
+at a maximum n/2 tiles before popping, so the memory usage will never surpass n + n/2.
 
```

Comment:
# 3

I _think_ you're correct, but  have a niggling doubt about when it can start popping. I wonder if there are times when it has to get to n-1?

Anyway, the principle is good :)